### PR TITLE
Update installer links and instructions

### PR DESCRIPTION
**Pull request by NeoTheThird**
Here's some updated information about the UBports Installer.

Fixes https://github.com/ubports/ubports-installer/issues/785

### CODE REVIEW

**Comment by NeoTheThird** [comment]
Maybe we wait a couple of days until the next release of the UBports Installer, that will fix the current issues on macOS and Windows. In the long run, what's your opinion on where to put documentation for the installer? We also have troubleshooting info in the readme, we should decide on one place. I don't mind either way.

**Comment by NeoTheThird** [comment]
Thanks, what's your opinion on where to put stuff?

**Comment by UniversalSuperBox** [comment]
If we want to give people a single place to look for installation instructions, I'd pick the docs.

**Comment by NeoTheThird** [comment]
Branch check is failing for some reason, but merge works, so we should be fine. I'm good to merge now, if you want to review again @UniversalSuperBox.

File: userguide/install.rst [hunk 1]
```diff
@@ -18,8 +18,95 @@ Start the installer and follow the on-screen instructions that will walk you thr
...
 
+Troubleshooting
+---------------
+
+If you need help, you can join UBports' support channels on `telegram <https://t.me/WelcomePlus>`_ or `matrix <https://matrix.to/#/!KwdniMNeTmClpgHkND:matrix.org?via=matrix.org&via=ubports.chat&via=disroot.org>`_ or ask a question `in the forum <https://forums.ubports.com/>` or on `askubuntu <https://askubuntu.com>`. If you believe that the UBports Installer is not working correctly, you can also `file a new issue <https://github.com/ubports/ubports-installer/issues/new>` to help us solve the problem.
+
+Connection lost during installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Try a different USB cable and a different USB port on your computer. Cheap and old cables tend to loose connection during the installation.
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: Lose, not loose

File: userguide/install.rst [hunk 2]
```diff
@@ -18,8 +18,95 @@ Start the installer and follow the on-screen instructions that will walk you thr
...
+
+Virtual Machine
+"""""""""""""""
+
+Make sure the virtual machine is allowed to access the USB port.
+
+Windows ADB drivers
+"""""""""""""""""""
+
+Install the `universal adb driver <http://adb.clockworkmod.com/>`__ and try again.
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: Maybe reboot after installing?

File: userguide/install.rst [hunk 3]
```diff
@@ -18,8 +18,95 @@ Start the installer and follow the on-screen instructions that will walk you thr
...
+"""""""""""""""""""""
+
+Run ``echo 0x2ae5 >> ~/.android/adb_usb.ini`` in the terminal and restart the installer. This does not affect the snap package.
+
+Missing udev-rules
+""""""""""""""""""
+
+If the device not detected on Linux and packages other than snap, you might be missing **udev-rules**.
+
+1. See if cat ``/etc/udev/rules.d/51-android.rules`` exists and contains the rules below. If not, add them to the file and run
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: If cat?

File: userguide/install.rst (hunk 1 above)
Comment (NeoTheThird) [reply userguide/install.rst]: Huh, the more you know :)

File: userguide/install.rst [hunk 4]
```diff
@@ -1,35 +1,112 @@
...
 
-.. note::
+If you're an experienced android developer and want to help us bring Ubuntu Touch to more devices, visit the :doc:`porting section </porting/introduction>`.
 
-    If your device is not detected automatically when using the snap installer, then unplug your device, open a terminal and enter the following command:
+Troubleshooting
+---------------
 
-    .. code-block:: text
+If you need help, you can join UBports' support channels on `telegram <https://t.me/WelcomePlus>`_ or `matrix <https://matrix.to/#/!KwdniMNeTmClpgHkND:matrix.org?via=matrix.org&via=ubports.chat&via=disroot.org>`_ or ask a question `in the forum <https://forums.ubports.com/>` or on `askubuntu <https://askubuntu.com>`. If you believe that the UBports Installer is not working correctly, you can also `file a new issue <https://github.com/ubports/ubports-installer/issues/new>` to help us solve the problem.
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: Some of these links are missing the final underscore and don't get built

File: userguide/install.rst [hunk 5]
```diff
@@ -1,35 +1,112 @@
...
+Run ``echo 0x2ae5 >> ~/.android/adb_usb.ini`` in the terminal and restart the installer. This does not affect the snap package.
+
+Missing udev-rules
+""""""""""""""""""
+
+If the device not detected on Linux and packages other than snap, you might be missing **udev-rules**.
+
+1. See if ``/etc/udev/rules.d/51-android.rules`` exists and contains the rules below. If not, add them to the file and run
+   ``sudo service udev restart`` or
+   ``sudo udevadm control --reload-rules && udevadm trigger``.
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: These do not appear on separate lines. Also, is `service udev restart` appropriate with systemd?

File: userguide/install.rst [hunk 6]
```diff
@@ -1,35 +1,112 @@
...
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="0fce", MODE="0666", GROUP="plugdev"
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="0930", MODE="0666", GROUP="plugdev"
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="19d2", MODE="0666", GROUP="plugdev"
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="2ae5", MODE="0666", GROUP="plugdev"
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="2a45", MODE="0666", GROUP="plugdev"
+
+Other issues
+^^^^^^^^^^^^
+
+If the trouble shooting tips do not work, you might also try following the manual installation instructions for your device `on the devices page <https://devices.ubuntu-touch.io/>`_.
```
Comment (UniversalSuperBox) [inline userguide/install.rst]: Troubleshooting is one word

File: userguide/install.rst (hunk 5 above)
Comment (NeoTheThird) [reply userguide/install.rst]: That's why the second command is there

**Pull request merged**